custom_components/unifiprotect/config_flow.py

```diff
@@ -90,13 +90,6 @@ async def async_step_user(self, user_input=None):
                 CONF_USERNAME: user_input.get(CONF_USERNAME),
                 CONF_PASSWORD: user_input.get(CONF_PASSWORD),
             },
-            options={
-                CONF_USERNAME: user_input.get(CONF_USERNAME),
-                CONF_PASSWORD: user_input.get(CONF_PASSWORD),
-                CONF_DISABLE_RTSP: False,
-                CONF_DOORBELL_TEXT: "",
-                CONF_SNAPSHOT_DIRECT: False,
-            },
```

Comment:
We should leave `CONF_DISABLE_RTSP`, `CONF_DOORBELL_TEXT `, and `CONF_SNAPSHOT_DIRECT` as these are options.  `CONF_USERNAME` and `CONF_PASSWORD` should be removed though since they are not

Reply:
I have added them back.